### DISCUSSION TITLE
Is there any way to transform AST to TypeScript code?

### DISCUSSION THREAD

**Discussion by azat-io** [typescript-eslint/typescript-eslint#7025] [Technical Discussions]
Something like [@babel/generator](https://babeljs.io/docs/babel-generator):

```js
const generate = require("@babel/generator");

Identifier(path) {
  console.log(generate(path.node).code); // code string
},
```

Or like this [davidbonnet/astring](https://github.com/davidbonnet/astring)

**Comment by JoshuaKGoldberg** [comment]
👋 The _Technical Discussions_ category of our GitHub Discussions is meant for _"talking about the architecture and/or APIs of typescript-eslint"_. Please use either [StackOverflow](https://stackoverflow.com) [our new Discord](https://discord.com/invite/FSxKq8Tdyg) if you just have a general question.

**Comment by bradzacher** [comment]
Also, duplicate of https://github.com/typescript-eslint/typescript-eslint/discussions/6377
